src/Protocol/Authenticator.cpp

```diff
@@ -51,6 +50,24 @@ void cAuthenticator::ReadSettings(cSettingsRepositoryInterface & a_Settings)
 	m_Server             = a_Settings.GetValueSet ("Authentication", "Server", DEFAULT_AUTH_SERVER);
 	m_Address            = a_Settings.GetValueSet ("Authentication", "Address", DEFAULT_AUTH_ADDRESS);
 	m_ShouldAuthenticate = a_Settings.GetValueSetB("Authentication", "Authenticate", true);
+
+	if (!cUrlParser::Validate(m_Server))
+	{
+		LOGWARNING("%s %d: Supplied invalid URL for authentication server: \"%s\", using default!", __FUNCTION__, __LINE__, m_Server.c_str());
```

Comment:
The error message should ideally point to the exact setting that is wrong, to help the users.